test/e2e/federation-ingress.go

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
This is a test for ingress resource in federation control plane?
Leaving this as is